rtCommon/fileWatcher.py

```diff
@@ -235,14 +234,20 @@ def __init__(self):
         self.waitLoopCount = 0
         # create a listening thread
         self.fileNotifyQ = Queue()  # type: None
-        self.notifier = inotify.adapters.Inotify()
-        self.notify_thread = threading.Thread(name='inotify', target=self.notifyEventLoop)
-        self.notify_thread.setDaemon(True)
-        self.notify_thread.start()
+        try:
+            testing_fd = inotify.calls.inotify_init()
```

Comment:
Maybe add a comment above this line so future developers will know why we did this. Something like:
'''For Docker linux containers running on Mac M1 chipset the inotify command fails. But we aren't able to catch the exception condition that occurs when creating an object using Inotify(), because the inotify.__del__ function has the exception in it.
So we add the dummy line below to test if inotify calls are working and this call will throw a catchable exception if inotify calls are not working.
'''